edxval/tests/test_api.py

```diff
@@ -1693,10 +1693,8 @@ def setUp(self):
         self.transcript_url = api.create_or_update_video_transcript(
             self.video_id,
             'ur',
-            'The_Arrow.srt',
-            TranscriptFormat.SRT,
-            provider=TranscriptProviderType.CUSTOM,
-            file_data=File(open(self.arrow_transcript_path)),
+            metadata={'file_format': TranscriptFormat.SRT},
```

Comment:
don't we need to pass the `provider` in the metadata?

Reply:
TranscriptProviderType.CUSTOM is default provider.